index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Quick startup</title>
+</head>
+
+<body>
+    <!--This wraps on my page together-->
+    <div class="container">
+        <header class="header-page">
+            <!--header contents goes here-->
+
+            <nav>
+                <!--This is my Logo-->
+                <a href=''>Solorics</a>
+
+                <!--Toggle button goes here-->
+                <img src="images/Union.png" alt="Toggle_Image_Button" class="toggle">
+            </nav>
+        </header>
+        <section class="main-page">
+            <!--main contents goes here-->
+            <div>
+                <h1>Opeyemi</h1>
+                <h2>Software Developer</h2>
+                <p>I can help you build a product , feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+               <hr>
+
+            </div>
+
+
+        </section>
+
+        <div class="section2">
+
```

Comment:
- [OPTIONAL] I think it would be better to use (semantic tags)  `<section class="section2"` instead of using `<div class="section2">` to follow [the HTML best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 👌💯

Reply:
Thanks will fix all in the next PR